src/components/menu/menu-item/menu-item.spec.js

```diff
@@ -419,4 +421,84 @@ describe("MenuItem", () => {
       });
     });
   });
+
+  describe("icon only menus and submenus", () => {
+    it("should render an icon into the menu item", () => {
+      wrapper = mount(
+        <MenuItem icon="settings" ariaLabel="Settings" keyboardOverride="s" />
+      );
+
+      expect(wrapper.find(StyledIcon).first().exists()).toBe(true);
+    });
+
+    it("should render an icon into the submenu item", () => {
+      wrapper = mount(
+        <MenuItem
+          icon="settings"
+          submenu
+          ariaLabel="Settings"
+          keyboardOverride="s"
+        >
+          <MenuItem icon="home" ariaLabel="Home" keyboardOverride="s" />
+        </MenuItem>
+      );
+
+      expect(wrapper.find(StyledIcon).first().exists()).toBe(true);
```

Comment:
There should only be one `StyledIcon` in the wrapper anyway here because the submenu isn't open, so the `.first()` isn't needed

Reply:
removed